pkg/universal-cnf/config/config.go

```diff
@@ -20,19 +20,30 @@ import (
 	"io/ioutil"
 	"os"
 	"os/exec"
+	"sync"
 
 	"github.com/davecgh/go-spew/spew"
 	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connection"
 	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connection/mechanisms/memif"
 	"github.com/networkservicemesh/networkservicemesh/sdk/client"
 	"github.com/sirupsen/logrus"
 	"go.ligato.io/vpp-agent/v3/proto/ligato/vpp"
+	"go.ligato.io/vpp-agent/v3/proto/ligato/vpp/interfaces"
 	"gopkg.in/yaml.v2"
 )
 
+// Is it possible to not use a global var?
+var PassThroughMemifs L2XconnMemifs
```

Comment:
It is best to avoid globals entirely. They make reading, modifiying and testing code harder, create tight coupling and also require extra work to make the variables safe for concurrent access.

Couldn't this be a field in the `UniversalCNFEndpoint`?

Reply:
It actually wasn't my intention, I think accidentally modified the last return statement without noticing it. Thank you for pointing it out